data/containers/autoyast_containers.xml.ep

```diff
@@ -39,13 +39,8 @@
     </mode>
   </general>
   <bootloader>
-    % unless ($check_var->('ARCH', 's390x')) {
-    <device_map config:type="list">
-      <device_map_entry>
-        <firmware>hd0</firmware>
-        <linux>/dev/vda</linux>
-      </device_map_entry>
-    </device_map>
+    % if ($check_var->('ARCH', 'ppc64le') or $check_var->('ARCH', 's390x')) {
+    <loader_type>grub2</loader_type>
```

Comment:
Why `device_map` and `loader_type` has been switched?

Reply:
To fix some syntax issues I encountered in this xml I had to deeply check&match each line with documentation. So when It comes to *loader_type* and *device_map*, looking in https://documentation.suse.com/sles/15-SP3/single-html/SLES-autoyast/index.html#CreateProfile-Bootloader I noted that basic order was inverted, so I aligned the code to the format suggested in the documentation reference,also to ease controls of the full syntax flow.

Reply:
Since the VRs are passing, I'd say this works, no?